src/tests/ftest/util/server_utils.py

```diff
@@ -378,6 +389,55 @@ def detect_engine_start(self, host_qty=None):
         # Define the expected states for each rank
         self._expected_states = self.get_current_state()
 
+    def get_detected_engine_count(self, sub_process):
+        """Get the number of detected joined engines.
+
+        Args:
+            sub_process (process.SubProcess): subprocess used to run the command
+
+        Returns:
+            int: number of patterns detected in the job output
```

Comment:
It looks like this method returns bool.

Reply:
Yes it does.  I'll fix it in a future PR.